adserver/auth/models.py

```diff
@@ -83,6 +83,14 @@ class User(AbstractBaseUser, PermissionsMixin):
     advertisers = models.ManyToManyField(Advertiser, blank=True)
     publishers = models.ManyToManyField(Publisher, blank=True)
 
+    # Notifications
+    notify_on_completed_flights = models.BooleanField(
+        default=True,
+        help_text=_(
+            "Receive an email notification when an advertising flight is complete"
+        ),
+    )
+
```

Comment:
I do worry this might get a bit overloaded as we add more features. Probably will need a checkbox HTML element & a ForeignKey or similar in the future. This seems fine for now until we have a few more notifications.